server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -132,6 +132,11 @@ public List<UploadedIndexMetadata> getIndices() {
         return indices;
     }
 
+    // TODO: Remove this once upload PR is merged.
```

Comment:
Is this waiting on a different change?

Reply:
Yes, it is waiting on upload PR: https://github.com/opensearch-project/OpenSearch/pull/10404. 

These methods are introduced in upload PR, once it is merged we will remove this TODO from there.